site/pages/index.tsx

```diff
@@ -16,7 +16,7 @@ export const getStaticProps: GetStaticProps<Props> = async (ctx) => {
   const translation = await loadTranslation(ctx.locale);
   const blockRate = await fetchBlockRate();
   const weeklyStakingRewards = await getStakingRewards({
```

Comment:
```suggestion
  const monthlyStakingRewards = await getStakingRewards({
```

Reply:
good catch, updated the translation as well.